release_dashboard/test/widgets/common/checkbox_substep_test.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:conductor_ui/widgets/common/checkbox_substep.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  group('Checkbox widget tests', () {
+    const String substepName = 'substep1';
+    const String substepSubtitle = 'subtitle1';
+    testWidgets('Displays elements correctly', (WidgetTester tester) async {
+      await tester.pumpWidget(
+        MaterialApp(
+          home: Material(
+            child: CheckboxAsSubstep(
+              isChecked: false,
+              clickCallback: () {},
+              substepName: substepName,
+              subtitle: const SelectableText(substepSubtitle),
+            ),
+          ),
+        ),
+      );
+
+      Finder checkboxListTile = find.byType(CheckboxListTile);
+      expect(checkboxListTile, findsOneWidget);
+      expect(find.text(substepName), findsOneWidget);
```

Comment:
can you check for all the substeps?

Reply:
Yea, this just checks the common widget of the checkbox substep itself only. All the checkbox substeps are checked [here](https://github.com/Yugue/cocoon/blob/6cf0fdbe82a5b2eaa70248aab88341bebff6eef6/release_dashboard/test/widgets/engine_cherrypicks_substeps_test.dart#L11).